ignite/pkg/jsonfile/jsonfile.go

```diff
@@ -201,12 +201,19 @@ func (f *JSONFile) Field(key string, param interface{}) error {
 }
 
 // WithKeyValue update a file value object by key
-func WithKeyValue(key string, value string) UpdateFileOption {
+func WithKeyValue(key, value string) UpdateFileOption {
 	return func(update map[string][]byte) {
 		update[key] = []byte(`"` + value + `"`)
 	}
 }
 
+// WithKeyValueByte update a file byte value object by key
+func WithKeyValueByte(key string, value []byte) UpdateFileOption {
+	return func(update map[string][]byte) {
+		update[key] = bytes.Join([][]byte{[]byte(`"`), value, []byte(`"`)}, []byte(""))
```

Comment:
Took me ages to read that xD

But the `append` alternative below is not a lot more readable so it's fine.

```go
update[key] = append([]byte(`"`), append(value, []byte(`"`)...)...)
```

Reply:
https://github.com/ignite/cli/pull/3208/commits/5963952c243a79413e4948f6c13ac1ba884969d4